Homework1/Task2.py

```diff
@@ -0,0 +1,11 @@
+# Задача 2: Найдите сумму цифр трехзначного числа.
+# *Пример:*
+
+# 123 -> 6 (1 + 2 + 3)
+# 100 -> 1 (1 + 0 + 0) |
+
+num = int(input("Введите трехзначное число: "))
+a = num % 10
+b = (num % 100) // 10
```

Comment:
У операций деления одинаковый приоритет.
```suggestion
b = num % 100 // 10
```

Reply:
Можно по-другому
```suggestion
b = num // 10 % 10
```